playbooks/common/openshift-cluster/upgrades/v3_3/master_config_upgrade.yml

```diff
@@ -48,3 +48,7 @@
     dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
     yaml_key: 'controllerConfig.servicesServingCert.signer.keyFile'
     yaml_value: service-signer.key
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'admissionConfig.pluginConfig'
+    yaml_value: "{{ openshift.master.kube_admission_plugin_config | to_padded_yaml(level=2) }}"
```

Comment:
`modify_yaml` should take care of the `to_padded_yaml` bits so we can remove that piece.

```
- modify_yaml:
    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
    yaml_key: 'admissionConfig.pluginConfig'
    yaml_value: "{{ openshift.master.kube_admission_plugin_config }}"
  when: "'kube_admission_plugin_config' in openshift.master"
```

But we're also completely overwriting the original `admissionConfig.pluginConfig` which may have had different config values.

Testing with these inventory vars you'll see the `ProjectRequestLimit` disappear after the upgrade.

```
openshift_master_admission_plugin_config={"ProjectRequestLimit":{"configuration":{"apiVersion":"v1","kind":"ProjectRequestLimitConfig","limits":[{"selector":{"admin":"true"}},{"maxProjects":"1"}]}},"PodNodeConstraints":{"configuration":{"apiVersion":"v1","kind":"PodNodeConstraintsConfig"}}}
openshift_master_kube_admission_plugin_config={"RunOnceDuration":{"configuration":{"apiVersion":"v1","kind":"RunOnceDurationConfig","activeDeadlineSecondsOverride":"3600"}},"ClusterResourceConfig":{"configuration":{"apiVersion":"v1","kind":"ClusterResourceOverrideConfig","limitCPUToMemoryPercent":"200","cpuRequestToLimitPercent":"6","memoryRequestToLimitPercent":"60"}},"PodNodeConstraints":{"configuration":{"apiVersion":"v1","kind":"PodNodeConstraintsConfig"}}}
```

Running byo/config after upgrade will restore the original `admissionConfig.pluginConfig` but removes the `kubernetesMasterConfig.admissionConfig` that moved over during upgrade.